src/construction.rs

```diff
@@ -1,8 +1,8 @@
-mod casting;
-mod constructors;
-mod rand_impl;
-mod serde_impl;
-mod serialization;
-mod to_primitive;
-
-pub(crate) use self::to_primitive::PrimitiveTy;
+mod casting;
```

Comment:
This git diff looks strange to me. What has changed here? Is it a preview bug?

Reply:
I think what is happening here is that the newlines were `\r\n` and all converted to `\n`. Next time I will make the newline conversion a separate commit